local_planner/src/nodes/common.h

```diff
@@ -90,7 +90,19 @@ void normalize(geometry_msgs::Point& p);
 **/
 double velocityLinear(double max_vel, double slope,
                       double v_old, double elapsed);
+/**
+* @brief     wrappes the input angle in to plus minus PI space
+* @param[in] angle to be wrapped  [rad]
+* @returns   angle wrapped [rad]
```

Comment:
this function doesn't return anything. instead write:
`@param[in,out] angle to be wrapped [rad]`